storage/google/cloud/storage/blob.py

```diff
@@ -1910,7 +1910,8 @@ def metadata(self, value):
         :type value: dict
         :param value: (Optional) The blob metadata to set.
         """
-        value = {k: str(v) for k, v in value.items()}
+        if value is not None:
+            value = {k: str(v) for k, v in value.items()}
```

Comment:
`value` has tag `Optional`, so it looks like regression
Do we need to do the same thing for bucket labels? I don't see `Optional` tag for them

Reply:
I think the documentation is incorrect. The code is correct in that the property metadata is optional but should not be optional when setting this value.

Reply:
Okay, deleted `Optional` tag. As I see, it was added by some kind of a script